lib/reporting/command_line_options.rb

```diff
@@ -27,12 +32,20 @@ def parse!(argv, out: STDOUT)
         end
 
         opts.on('--week=DATE', <<~STR.squish) do |date_v|
-          run the report for the week (Sun-Sat) that includes the date in YYYY-MM-DD format'
+          run the report for the week (Sun-Sat) that includes the date in YYYY-MM-DD format
         STR
           date = Date.parse(date_v)
           period = :week
         end
 
+        opts.on('--month=DATE', <<~STR.squish) do |date_v|
+          run the report for the month that includes the date in YYYY-MM-DD format. recommended
+          to include `--threads 10` and `--slice 1h` to reduce likelihood of timeout
```

Comment:
[feedback wanted] instead of adding `--threads` and `--slice` flags, i considered just made internal changes if the `--month` flag was present, but i liked the ability for a user to have that granularity. interested in feedback if anyone disagrees with that decision!

Reply:
I think we can have both! I feel like it would make sense to change the default slice size for month vs week, and if an explicit slice size is provided, that would take precedence.

Reply:
I like giving the user the choice. Doesn't hurt anything to give the choice, and if its not something super important they can choose to not run a bunch of threads to give the boxes a bit of a break while pulling all the info.